.github/workflows/node.js.yml

```
@@ -27,5 +27,12 @@ jobs:
         node-version: ${{ matrix.node-version }}
         cache: 'npm'
     - run: npm ci
-    - run: npm run build --if-present
-    - run: npm test
+    - run: npm run build
+    - run: npm test:coverage
+    - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v4
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 33: mapping values are not allowed in this context

(syntax-check)